packages/swap-sdk-core/src/fractions/fraction.ts

```diff
@@ -125,7 +125,7 @@ export class Fraction {
   public toSignificant(
     significantDigits: number,
     format: object = { groupSeparator: '' },
-    rounding: Rounding = Rounding.ROUND_HALF_UP
+    rounding: Rounding = Rounding.ROUND_DOWN
```

Comment:
Can we add an additional rounding param here?
https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/utils/formatFractions.ts#L7

And then use Rouding.ROUND_DOWN for `formatAmount`
https://github.com/pancakeswap/pancake-frontend/blob/develop/packages/utils/formatFractions.ts#L24

Because using round half up for fraction formatting seems reasonable to me. What we need is to round down for `CurrencyAmount`